samples/calculator/iosApp/calculator.xcodeproj/project.pbxproj

```diff
@@ -86,70 +84,13 @@
 			path = calculator;
 			sourceTree = "<group>";
 		};
-		7FF94ADA2058464C00590D0D /* KotlinArithmeticParser */ = {
-			isa = PBXGroup;
-			children = (
-				7FF94AE42058485300590D0D /* src */,
-			);
-			name = KotlinArithmeticParser;
-			sourceTree = "<group>";
-		};
 		7FF94AE22058483900590D0D /* Frameworks */ = {
 			isa = PBXGroup;
 			children = (
 			);
 			name = Frameworks;
 			sourceTree = "<group>";
 		};
-		7FF94AE42058485300590D0D /* src */ = {
-			isa = PBXGroup;
-			children = (
-				7FF94AE52058485300590D0D /* main */,
-			);
-			name = src;
-			path = ../common/src;
-			sourceTree = SOURCE_ROOT;
-		};
-		7FF94AE52058485300590D0D /* main */ = {
-			isa = PBXGroup;
-			children = (
-				7FF94AE62058485300590D0D /* kotlin */,
-			);
-			path = main;
-			sourceTree = "<group>";
-		};
-		7FF94AE62058485300590D0D /* kotlin */ = {
-			isa = PBXGroup;
-			children = (
-				7FF94AE72058485300590D0D /* org */,
-			);
-			path = kotlin;
-			sourceTree = "<group>";
-		};
-		7FF94AE72058485300590D0D /* org */ = {
-			isa = PBXGroup;
-			children = (
-				7FF94AE82058485300590D0D /* konan */,
-			);
-			path = org;
-			sourceTree = "<group>";
-		};
-		7FF94AE82058485300590D0D /* konan */ = {
-			isa = PBXGroup;
-			children = (
-				7FF94AE92058485300590D0D /* arithmeticparser */,
-			);
-			path = konan;
-			sourceTree = "<group>";
-		};
-		7FF94AE92058485300590D0D /* arithmeticparser */ = {
```

Comment:
Do I understand correctly that Xcode will no longer show Kotlin sources in project structure?

Reply:
Xcode can show any files in the project structure. It doesn't apply any syntax highlighting to Kotlin sources, but AppCode did (at least with Kotlin/Native plugin).

Reply:
There is no _updated_ Kotlin/Native plugin for AppCode. So let's put AppCode aside for now. I will check how to show Kotlin files in Xcode structure.

Reply:
Fixed.